src/common/types/core.js

```diff
@@ -302,6 +302,7 @@ type SharedLinkFeatures = {
 
 type BoxItem = {
     allowed_shared_link_access_levels?: Array<Access>,
+    archive_type?: 'archive' | 'archived-content',
```

Comment:
Nit: I think this would make it consistent with other places
```suggestion
    archive_type?: 'archive' | 'archive-content',
```

Reply:
Thanks for noticing that, I've fixed it, but backend response changed form, so I've updated this to new form